packages/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "allowSyntheticDefaultImports": true,
+    // "isolatedModules": true,
+    "jsx": "preserve",
+    "module": "ES2020",
+    "moduleResolution": "node",
+    "noUnusedLocals": true,
+    "strict": true,
+    "target": "ES2020",
+    "types": ["@emotion/react/types/css-prop", "@types/jest"],
```

Comment:
Is this it necessary to specify the `css-prop` types? We are already adding this reference to the top of each component index file:

```
///<reference types="@emotion/react/types/css-prop" />
```

This ensures consuming apps don't need to add the `css-prop` type to their `tsconfig`

Reply:
ah nice, i didn't realise that. how does that work when it's bundled?

Reply:
> Triple-slash directives are only valid at the top of their containing file.

should we make sure we prepend it to the build files?

Reply:
> how does that work when it's bundled
> should we make sure we prepend it to the build files

I don't think it matter when it's bundled, because the output is JS. It's only useful in TS files